### ISSUE TITLE
moodle_cmd_exec exploit shows Exploit failed: REXML::ParseException No close tag for /div

### ISSUE THREAD

**Issue by srikanthmalla** [rapid7/metasploit-framework#6879]
msf:
Framework: 4.11.4-2015071403
Console  : 4.11.4-2015071403.15168

When every option is set and exploit is pressed then the terminal shows:
[*] Started reverse double handler
[*] Authenticating as user: xxxxxxx
[*] Getting session key to update spellchecker if no session key was specified
[-] Exploit failed: REXML::ParseException No close tag for /div
Line: 1
Position: 29
Last 80 unconsumed characters:

[*] Exploit completed, but no session was created.

Please help me with this situation

**Comment by wchen-r7** [comment]
@srikanthmalla I might be able to help you, because it looks like the parser could be improved. But I am not able to find the installer for Moodle 2.5.2 or 2.2.3? Mind help?

**Comment by wchen-r7** [comment]
Never mind, I found it: https://download.moodle.org/stable25/

**Comment by wchen-r7** [comment]
I tested the module against moodle-2.5.2.zip, and it didn't really work for me.

The only thing I can tell that is wrong is the path the exploit is requesting to trigger the payload:

```
/lib/editor/tinymce/tiny_mce/3.4.9/plugins/spellchecker/rpc.php
```

On my test box, I don't have this path. But I have this:

```
/lib/editor/tinymce/plugins/spellchecker/rpc.php
```

It also looks like we didn't actually create a PR before landing to master, looks like it was added [directly to master](https://github.com/rapid7/metasploit-framework/commit/5b7694776738d29a2bc752d29e6f3cbcc2aa93a8), so there is no history on github documenting how it was tested/verified.

@srikanthmalla I can probably fix the bug you're seeing, but I can't really guarantee how well the exploit will work for you.
